app/controllers/concerns/idv/ab_test_analytics_concern.rb

```diff
@@ -16,7 +16,7 @@ def ab_test_analytics_buckets
         buckets = buckets.merge(lniv_args)
       end
 
-      buckets.merge(acuant_sdk_ab_test_analytics_args)
+      { ab_test_buckets: buckets.merge(acuant_sdk_ab_test_analytics_args) }
```

Comment:
+1 to moving these under their own key. I have a slight preference for `ab_tests` (since characters are so expensive these days)

Reply:
(Or even `tests`, since they're not even always A/B tests are they?)

Reply:
`tests` seems too generic to me, but `ab_tests` is better (even if it may be an a/b/c/d test)